### PR TITLE
Fix error in getting array element type

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

I discovered that array elements were returning TypeUnknown for a LabVIEW service. This was a mistake in earlier refactoring.

I refactored the logic between Get Metadata from Configurations and Get Metadata from Results so they re-use the code that fetches the parameter information from the cluster element.

### Why should this Pull Request be merged?

Fixes LabVIEW services with arrays. Simplifies logic in Get Metadata.

### What testing has been done?

I'll build the packages and test with them to ensure what I was seeing is fixed.

### CODE REVIEW

File: Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Configurations.vi
Comment (dixonjoel) [inline Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Configurations.vi:1]: Before:
image

File: Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Results.vi
Comment (dixonjoel) [inline Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Results.vi:1]: Before:
image

File: Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Configurations.vi
Comment (dixonjoel) [reply Source/Runtime/MeasurementLink Measurement Server/Helpers/Get MetaData from Configurations.vi:1]: After:
image

File: Source/Runtime/MeasurementLink Measurement Server/Helpers/Get Parameter Information.vi
Comment (dixonjoel) [inline Source/Runtime/MeasurementLink Measurement Server/Helpers/Get Parameter Information.vi:1]: image

Comment (pbirkhol-ni) [reply Source/Runtime/MeasurementLink Measurement Server/Helpers/Get Parameter Information.vi:1]: The danger in using variants. Not saying we have a choice here, but I've done similar things many times. The coercion dot has usually alerted me.

**Comment by dixonjoel** [comment]
> I wish we had more tests in areas like this. This code is definitely testable. I am not going to require tests to submit this PR, but if you want to take a shot at writing some tests, I won't stop you. :)

How do we handle the fact that these are private VIs inside the library when we write tests?

**Comment by pbirkhol-ni** [comment]
> > I wish we had more tests in areas like this. This code is definitely testable. I am not going to require tests to submit this PR, but if you want to take a shot at writing some tests, I won't stop you. :)
> 
> @pbirkhol-ni How do we handle the fact that these are private VIs inside the library when we write tests?

Make them community scope and then add the test library as a friend.

**Pull request merged**